Source/Diagnostics/ReducedDiags/FieldProbe.cpp

```diff
@@ -362,60 +420,81 @@ void FieldProbe::ComputeDiags (int step)
 
                 // this check is here because for m_field_probe_integrate == True, we always compute
                 // but we only write when we truly are in an output interval step
-                if (m_intervals.contains(step+1)) {
-                    for (int ip = 0; ip < np; ip++) {
+                if (m_intervals.contains(step+1))
+                {
+                    for (auto ip=numparticles - np; ip < numparticles; ip++) 
```

Comment:
Ups, I think that was wrong.
Keep
```suggestion
                    for (auto ip=0; ip < np; ip++) 
```
and below add the offset for `m_data`